doc/utstring.txt

```diff
@@ -154,16 +154,18 @@ Operations
 | utstring_new(s) | allocate a new utstring
 | utstring_renew(s) | allocate a new utstring (if s is `NULL`) otherwise clears it
 | utstring_free(s) | free an allocated utstring
-| utstring_init(s)  |  init a utstring (non-alloc)
-| utstring_done(s) |  dispose of a utstring (non-allocd)
+| utstring_init(s)  | init a utstring (non-alloc)
+| utstring_done(s) | dispose of a utstring (non-alloc)
```

Comment:
Thanks! I'll take these diffs to the existing docs upstream right now (hopefully in a way that merges cleanly with this PR).